jme3-android-examples/jme3-android-tests/app/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "org.jmonkeyengine.jme3androidexamples"
+        minSdkVersion 22
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    configurations.all {
+        exclude group:"org.jmonkeyengine",module:"jme3-desktop"
+        exclude group: 'jbullet', module:'jbullet'
+        exclude group: 'stack-alloc', module:'stack-alloc'
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.google.android.material:material:1.3.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    /*add the JMonkeyEngine Dependencies*/
+    runtimeOnly project(":stack-alloc")
+    implementation project(":jbullet")
+    implementation "org.jmonkeyengine:jme3-core:3.4.0-alpha7"
```

Comment:
why this is changed from to `compile project(':jme3-core')` implementation "org.jmonkeyengine:jme3-core:3.4.0-alpha7"?

the same goes for other dependencies as well.

Reply:
Yep, it's because i have separated this android-example in a separate module(package) from jmonkeyengine, that has it's own gradle builds & source, that was the only way to make it runnable, because Android gradle is different from java gradle, btw this example wasn't runnable & there were build files that are missing, listed in the title of the PR.